x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/start_step.tsx

```diff
@@ -128,6 +172,14 @@ export const StartStep: React.FC<StartStepProps> = ({
                   'xpack.enterpriseSearch.createConnector.startStep.euiFormRow.descriptionLabel',
                   { defaultMessage: 'Description' }
                 )}
+                labelAppend={
```

Comment:
Do we generally specify when something is optional?

Reply:
That's a good question. We have discussed this topic several times with no commitment yet. This specific change was done based on Serena's feedback. IMO signposting on small forms what can be skipped would make quicker go to the next step, which should be the goal, create the connector with the less amount of clicks and interactions as possible.